assets/queries/ansible/azure/public_storage_account/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "Public Storage Account",
   "severity": "HIGH",
   "category": "Access Control",
-  "descriptionText": "Check if 'network_acls' is open to public.",
+  "descriptionText": "Storage Account should not be public",
```

Comment:
This description could have more details about the impact. Do you feel comfortable trying to improve it?